syllabus/syllabus-app/src/webapp/syllabus/delete_confirm.jsp

```diff
@@ -21,7 +21,7 @@
 		  	<h3>
 					<h:outputText  value="#{msgs.delConfNotice}" />
 			</h3>	
-			<div class="alertMessage">
+			<div class="sak-banner-error">
```

Comment:
For confirmations, let's use the sak-banner-warn class so it is a warning rather than an error. Thanks.